enterprise/server/workflowconf/workflowconf.go

```diff
@@ -0,0 +1,49 @@
+package workflowconf
```

Comment:
maybe we ought to make workflow a "container" directory and move workflowconf -> workflow/conf, workflowcmd -> workflow/command, and workflow -> workflow/runner?

Reply:
SGTM